styles/components/_hero.scss

```diff
@@ -0,0 +1,47 @@
+.hero {
+  background: $tundora 100% / cover;
```

Comment:
This is a pretty visible area, the color that is present before the image downloads. Probably worth pulling out to constants.scss